nucypher/blockchain/eth/sol/source/contracts/MinersEscrow.sol

```diff
@@ -166,9 +165,8 @@ contract MinersEscrow is Issuer {
     * @notice Set mining adjudicator address
     **/
     function setMiningAdjudicator(MiningAdjudicatorInterface _miningAdjudicator) external onlyOwner {
-        // Three-part require...
+        // Two-part require...
         require(address(miningAdjudicator) == address(0) &&  // Can't adjudicator once it is set.
```

Comment:
``` suggestion
        require(address(miningAdjudicator) == address(0) &&  // Can't transact adjudicator once address is set.
```

Is this what you mean here?

Reply:
I modified the comment a little bit and put it to `require` message